src/krisi/evaluate/metric.py

```diff
@@ -80,78 +87,47 @@ def __str__(self) -> str:
     def __repr__(self) -> str:
         return print_metric(self, repr=True)
 
-    def evaluate(self, y: Targets, predictions: Predictions) -> None:
-        assert self.func is not None, "`func` has to be set to calculate Metric result."
-        check_valid_pred_target(y, predictions)
-
+    def evaluation_(self, *args) -> "Metric":
```

Comment:
is this a mutable thing?
I only encountered underscore at the end of a property when it was a result of a calculation. if it's a more "private-like" method, the underscore usually goes to the beginning

Reply:
Yes, it mutates the object. It was previously also private (with dashes in front), but I had to access it from `Group`

Reply:
Then its perfect! Because it shouldn't be called by the user but the method should still be exposed.

Reply:
I mean a single leading dash :) but as you wish!

Reply:
Done